observationtree/src/main/scala/explore/observationtree/TargetObsList.scala

```diff
@@ -226,91 +226,90 @@ object TargetObsList {
         UndoRegion[TargetsWithObs] { undoCtx =>
           DragDropContext(onDragEnd = onDragEnd(undoCtx.setter))(
             <.div(
-              props.targetsWithObs.get.targets.elements.toTagMod {
-                target =>
-                  val targetId = target.id
-
-                  val targetObs = obsByTarget.getOrElse(targetId, List.empty)
-                  val obsCount  = targetObs.length
-
-                  val opIcon =
-                    targetObs.nonEmpty.fold(
-                      Icon(
-                        "chevron " + state.collapsedTargetIds
-                          .exists(_ === targetId)
-                          .fold("right", "down")
-                      )(^.cursor.pointer,
-                        ^.onClick ==> { e: ReactEvent =>
-                          e.stopPropagationCB >> toggleCollapsed(targetId)
-                        }
-                      ),
-                      Icon("chevron right")
-                    )
-
-                  Droppable(target.id.toString) {
-                    case (provided, snapshot) =>
-                      <.div(
-                        provided.innerRef,
-                        provided.droppableProps,
-                        getListStyle(snapshot.isDraggingOver)
+              props.targetsWithObs.get.targets.elements.toTagMod { target =>
+                val targetId = target.id
+
+                val targetObs = obsByTarget.getOrElse(targetId, List.empty)
+                val obsCount  = targetObs.length
+
+                val opIcon =
+                  targetObs.nonEmpty.fold(
+                    Icon(
+                      "chevron " + state.collapsedTargetIds
+                        .exists(_ === targetId)
+                        .fold("right", "down")
+                    )(^.cursor.pointer,
+                      ^.onClick ==> { e: ReactEvent =>
+                        e.stopPropagationCB >> toggleCollapsed(targetId)
```

Comment:
The other day I discovered
https://github.com/japgolly/scalajs-react/blob/master/core/src/main/scala/japgolly/scalajs/react/Callback.scala#L758

Don't know if it is useful

Reply:
Interesting. Good tip.